.github/workflows/ci.yml

```diff
@@ -125,14 +125,11 @@ jobs:
         env:
           PULUMI_PYTHON_CMD: python
           TESTPARALLELISM: 3
-          SKIPPED_TESTS: "alicloud,digitalocean,kubernetes,openstack,equinix-metal,civo,aiven,auth0,github,oci,java-jbang,java-gradle,azuredevops,container"
```

Comment:
Not sure why these are overridden -- I believe the steps should inherit values set on line 22.